### PR TITLE
Update incorrect licensing statement

### PR DESCRIPTION
**Pull request by Weishaupt**
According to Footnote 2 on the [Exchange Online Service Limits](https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits#storage-limits) shared mailboxes require a license in three defined cases.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/email/remove-license-from-shared-mailbox.md [hunk 1]
```diff
@@ -25,7 +25,14 @@ description: "Remove license from a shared mailbox to assign it to another user.
 
 # Remove a license from a shared mailbox
 
-Shared mailboxes don't need a license unless the mailbox has over 50GB of data. Follow these instructions to remove a license from a shared mailbox so that you can either assign it to a user or return the license so that you aren't paying for a license you don't need.
+Shared mailboxes usually don't require a license. Follow these instructions to remove a license from a shared mailbox so that you can either assign it to a user or return the license so that you aren't paying for a license you don't need.
+
+> [!NOTE]
+> A license is required in the following scenarios
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/remove-license-from-shared-mailbox.md]:
```suggestion
> A license is required in the following scenarios:
```

File: microsoft-365/admin/email/remove-license-from-shared-mailbox.md [hunk 2]
```diff
@@ -25,7 +25,14 @@ description: "Remove license from a shared mailbox to assign it to another user.
...
+> 1. The shared mailbox has more than 50 GB of storage in use.
+> 2. The shared mailbox uses in-place archiving
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/remove-license-from-shared-mailbox.md]:
```suggestion
> 2. The shared mailbox uses in-place archiving.
```

File: microsoft-365/admin/email/remove-license-from-shared-mailbox.md [hunk 3]
```diff
@@ -25,7 +25,14 @@ description: "Remove license from a shared mailbox to assign it to another user.
...
+> 3. The shared mailbox is placed in litigation hold
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/remove-license-from-shared-mailbox.md]:
```suggestion
> 3. The shared mailbox is placed in litigation hold.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@Weishaupt - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by Weishaupt** [comment]
> 
> 
> @Weishaupt - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

Done.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@pebaum - PR has been copy edited and is ready for final review.  

Thank you.

cc: @kenwith

**Pull request merged**